protocol/v1/qbft/instance/change_round.go

```diff
@@ -18,8 +18,14 @@ import (
 
 // ChangeRoundMsgPipeline - the main change round msg pipeline
 func (i *Instance) ChangeRoundMsgPipeline() pipelines.SignedMessagePipeline {
+	validationPipeline := i.ChangeRoundMsgValidationPipeline()
```

Comment:
if tests are passing, why we need to align this error?

Reply:
Because I don't like how the below looks like 
```go
	case specqbft.ProposalMsgType:
		if err := i.ProposalMsgPipeline().Run(msg); err != nil {
			if errors.Is(err, signedmsg.ErrWrongRound) {
				// NOTE: These 4 checks will be replaced by one in a future PR.
				i.Logger.Debug(fmt.Sprintf("message round (%d) does not equal state round (%d)", msg.Message.Round, i.State().GetRound()))
			}
			return false, fmt.Errorf("invalid proposal message: %w", err)
		}
	case specqbft.PrepareMsgType:
		if err := i.PrepareMsgPipeline().Run(msg); err != nil {
			if errors.Is(err, signedmsg.ErrWrongRound) {
				// NOTE: These 4 checks will be replaced by one in a future PR.
				i.Logger.Debug(fmt.Sprintf("message round (%d) does not equal state round (%d)", msg.Message.Round, i.State().GetRound()))
			}
			return false, err
		}
	case specqbft.CommitMsgType:
		if err := i.CommitMsgPipeline().Run(msg); err != nil {
			if errors.Is(err, signedmsg.ErrWrongRound) {
				// NOTE: These 4 checks will be replaced by one in a future PR.
				i.Logger.Debug(fmt.Sprintf("message round (%d) does not equal state round (%d)", msg.Message.Round, i.State().GetRound()))
			}
			return false, err
		}
	case specqbft.RoundChangeMsgType:
		if err := i.ChangeRoundMsgPipeline().Run(msg); err != nil {
			if errors.Is(err, signedmsg.ErrWrongRound) {
				// NOTE: These 4 checks will be replaced by one in a future PR.
				i.Logger.Debug(fmt.Sprintf("message round (%d) does not equal state round (%d)", msg.Message.Round, i.State().GetRound()))
			}
			return false, fmt.Errorf("invalid round change message: %w", err)
		}
```
		
In two cases I return just `err`, in other two a wrapped error, I did that just to pass tests, but in this PR I want to have the message processing same for all types, so I moved wrapped error inside. In my opinion, it would be much easier to understand.